.github/workflows/release.yaml

```diff
@@ -0,0 +1,117 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+env:
+  GO_VERSION: '1.23'
+  CGO_ENABLED: 0
+  VERSION: ${{ github.ref_name }}
+  TRAEFIKER_EMAIL: "traefiker@traefik.io"
+  CODENAME: mimolette
+
+jobs:
+
+  build-webui:
+    if: github.ref_type == 'tag'
+    uses: ./.github/workflows/template-webui.yaml
+
+  build:
+    if: github.ref_type == 'tag'
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ linux-amd64, linux-386, linux-arm, linux-arm64, linux-ppc64le, linux-s390x, linux-riscv64, darwin, windows-amd64, windows-arm64, windows-386, freebsd, openbsd ]
+    needs:
+      - build-webui
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v5
+        env:
+          ImageOS: ${{ matrix.os }}
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Install go releaser
+        run: go install github.com/goreleaser/goreleaser/v2@latest
+
+      - name: Artifact webui
+        uses: actions/download-artifact@v4
+        with:
+          name: webui.tar.gz
+
+      - name: Untar webui
+        run: |
+          tar xvf webui.tar.gz
+          rm webui.tar.gz
+
+      - name: Build
+        run: |
+          go generate
+          goreleaser release --clean -p 2 --timeout="90m" --config "$(go run ./internal/release "${{ matrix.os }}")"
+
+      - name: Artifact binaries
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ matrix.os }}-binaries
+          path: |
+            dist/**/*_checksums.txt
+            dist/**/*.tar.gz
+            dist/**/*.zip
+          retention-days: 1
+
+  release:
+    if: github.ref_type == 'tag'
+    runs-on: ubuntu-latest
+
+    needs:
+      - build
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Retrieve the secret and decode it to a file
+        env:
+          TRAEFIKER_RSA: ${{ secrets.TRAEFIKER_RSA }}
+        run: |
+          mkdir -p ~/.ssh
+          echo ${TRAEFIKER_RSA} | base64 --decode > ~/.ssh/traefiker_rsa
+
+      - name: Download All Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: dist/
+          pattern: "*-binaries"
+          merge-multiple: true
+
+      - name: Publish Release
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          cat dist/**/*_checksums.txt >> "dist/traefik_${VERSION}_checksums.txt"
+          rm dist/**/*_checksums.txt
```

Comment:
Maybe I missed something, but the [default shell](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) with run is bash. AFAIK, this syntax does not work with bash.

Reply:
It seems to work:
```
ll -R .
.:
total 16
drwxrwxr-x  4 youkoulayley youkoulayley 4096 nov.  29 09:39 ./
drwxr-x--- 48 youkoulayley youkoulayley 4096 nov.  29 09:39 ../
drwxrwxr-x  2 youkoulayley youkoulayley 4096 nov.  29 09:40 titi/
drwxrwxr-x  2 youkoulayley youkoulayley 4096 nov.  29 09:40 toto/

./titi:
total 8
drwxrwxr-x 2 youkoulayley youkoulayley 4096 nov.  29 09:40 ./
drwxrwxr-x 4 youkoulayley youkoulayley 4096 nov.  29 09:39 ../
-rw-rw-r-- 1 youkoulayley youkoulayley    0 nov.  29 09:40 titi_checksums.txt

./toto:
total 8
drwxrwxr-x 2 youkoulayley youkoulayley 4096 nov.  29 09:40 ./
drwxrwxr-x 4 youkoulayley youkoulayley 4096 nov.  29 09:39 ../
-rw-rw-r-- 1 youkoulayley youkoulayley    0 nov.  29 09:40 toto_checksums.txt
```

```
youkoulayley@Morgott:~$ rm dist/**/*_checksums.txt
youkoulayley@Morgott:~$ cd dist
youkoulayley@Morgott:~/dist$ ll -R .
.:
total 16
drwxrwxr-x  4 youkoulayley youkoulayley 4096 nov.  29 09:39 ./
drwxr-x--- 48 youkoulayley youkoulayley 4096 nov.  29 09:39 ../
drwxrwxr-x  2 youkoulayley youkoulayley 4096 nov.  29 09:42 titi/
drwxrwxr-x  2 youkoulayley youkoulayley 4096 nov.  29 09:42 toto/

./titi:
total 8
drwxrwxr-x 2 youkoulayley youkoulayley 4096 nov.  29 09:42 ./
drwxrwxr-x 4 youkoulayley youkoulayley 4096 nov.  29 09:39 ../

./toto:
total 8
drwxrwxr-x 2 youkoulayley youkoulayley 4096 nov.  29 09:42 ./
drwxrwxr-x 4 youkoulayley youkoulayley 4096 nov.  29 09:39 ../
 
```

Reply:
My bad not working with subfolder inside titi, toto etc.

Reply:
I added a suggestion